src/current/_includes/cockroachcloud/create-cluster-api-dedicated.md

```diff
@@ -0,0 +1,96 @@
+**API request**
+
+To create a **Dedicated** cluster, send a `POST` request to the `/v1/clusters` endpoint, specifying the following parameters:
+
+- `name`: your cluster's name, a short string with no whitespace.
+- `provider`: "GCP", "AWS", or`AZURE`. Note that support for Azure is in [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html).
+- `hardware`:
+  - `machine_spec`
+    - `machine_type`: machine type, e.g. "n2-standard-2".
+  - `storage_gib`: (int) storage in GiB.
+- `region_nodes`: a hash where each key is a region name and each value is the desired number of nodes (if not 0).
+- `spend_limit`: specified in dollars
+- `network_visibility`: PRIVATE or PUBLIC
+- `restrict_egress_traffic`: true or false
+
+[API reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters)
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+--url 'https://management-staging.crdb.io/api/v1/clusters' \
```

Comment:
Is it ok to leave in the URL for the staging clusters? I may not be understanding correctly.